general/clients/kodi.md

```diff
@@ -3,51 +3,58 @@ uid: clients-kodi
-> This will keep your media libraries up to date without waiting for a periodic re-sync from Kodi.
+# Add-on Repository
 
-> [!CAUTION]
-> Remote Kodi databases, like MySQL, are not supported. A local SQLite database is required (this is the default).
-
-## Installation Process
+## Install Add-on Repository
 
-### Install Add-On Repository
-
-The recommended install method is to use the official Jellyfin add-on repository.  This allows for easy install of the Jellyfin for Kodi add-on, as well as automatically keeping the add-on up to date with the latest version.  Any other Jellyfin related add-ons that may be built in the future will also be available in this repository.
+The most convenient install method of our Jellyfin add-ons is to use the official Kodi Jellyfin Repository.  Using this repository allows for easy install of our add-ons, as well as automatically keeping the add-ons up to date with the latest version.  Any other Jellyfin related add-ons that may be built in the future will also be available here.
 
 The installation method for the repository varies depending on what kind of device you're using, outlined below.
 
-#### General Use Devices (PCs and Tablets)
+### General Use Devices (PCs and Tablets)
 
 1. Download the repository installer found [here](https://kodi.jellyfin.org/repository.jellyfin.kodi.zip).
     * It will be saved as `repository.jellyfin.kodi.zip`
 2. Install the Jellyfin repository.
-    * Open Kodi, go to the settings menu,  and navigate to "Add-On Browser"
+    * Open Kodi, go to the settings menu,  and navigate to "Add-on Browser"
     * Select "Install from Zip File"
-        * If prompted, enter settings and enable "Unknown Sources", then go back to the Add-On Browser
+        * If prompted, enter settings and enable "Unknown Sources", then go back to the Add-on Browser
     * Select the newly downloaded file and it will be installed
-3. Proceed to [Install Jellyfin for Kodi Add-On](xref:clients-kodi#install-jellyfin-for-kodi-add-on)
 
-#### "Embedded" Devices (Android TV, Firestick, and other TV Boxes)
+### "Embedded" Devices (Android TV, Firestick, and other TV Boxes)
 
 1. Open Kodi, go to the settings menu, and navigate to "File manager"
     * Select "Add source"
     * In the text box, enter `https://kodi.jellyfin.org`
     * Enter a name for the data source, such as "Jellyfin Repo" and select Ok
-2. From the settings menu, navigate to "Add-On Browser"
+2. From the settings menu, navigate to "Add-on Browser"
     * Select "Install from Zip File"
-        * If prompted, enter settings and enable "Unknown Sources", then go back to the Add-On Browser
+        * If prompted, enter settings and enable "Unknown Sources", then go back to the Add-on Browser
     * Select the data source you just added
     * Install `repository.jellyfin.kodi.zip`
-3. Proceed to [Install Jellyfin for Kodi Add-On](xref:clients-kodi#install-jellyfin-for-kodi-add-on)
 
-### Install Jellyfin for Kodi Add-On
+# Jellyfin for Kodi
```

Comment:
linter failed here, can only have one # Header

Reply:
And I take offense to that, because they're 3 distinct topics and it seems stupid to basically indent everything one level for the lolz.